woocommerce/woocommerce-android
# Replace deprecated menu usages

Closes: #7806
Closes: #7807
Closes: #7808 
Closes: #7809


### Description

Replaces deprecated menu usages in:
* ProductImagesFragment
* ProductListFragment
* ProductSelectionListFragment
* AddProductCategoryFragment

### Testing instructions

* Open product details. Click on an image. Try to select or drag an image. Notice that menu item there and they do work
* Open product list. Notice that the search menu item works
* Open product list. Click on add product. Add group product. Add product to the group. Add product. Notice that menu item there and they do work
* Open product details. Add more details. Categories. Add category. Notice that the done menu item works
